patches/0005-Add-CNG-crypto-backend.patch

```diff
@@ -467,6 +467,18 @@ index 00000000000000..3d3d13709de5ac
 +func VerifyEd25519(pub *PublicKeyEd25519, message, sig []byte) error {
 +	panic("cryptobackend: not available")
 +}
++
++func SupportsDSA(l, n int) bool {
++	return n != 224
```

Comment:
It seems unusual to me to support all keys sizes *except* for one... add a comment? Or add a `cng.SupportsDSA` function to the CNG backend repo and call that to let it decide and document this instead?

Reply:
Agree that this deserve a comment. Having `cng.SupportsDSA` might be an overkill for now.